Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.8
+
+EXPOSE 5000
+
+COPY ./requirements.txt /app/requirements.txt
```

Comment:
Is this needed?
We have `COPY . /app` below anyway

Reply:
Copying just requirements.txt first lets us leverage Docker's cache